### PR TITLE
Display the number of members as a badge next to the title

### PR DESCRIPTION
**Pull request by neves**
I think it would be nice to know how many members are ;)
image

### CODE REVIEW

**Comment by vyper** [comment]
@neves Can you post a screenshot with the change? (:

File: app/views/welcome/index.html.slim [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 h3
   = t(".members")
+  .badge =< @members.size
```
Comment (vyper) [inline app/views/welcome/index.html.slim]: `=<` what it does?

Comment (neves) [inline app/views/welcome/index.html.slim]: Sure. Do you know an easy way to do that?

Comment (vyper) [inline app/views/welcome/index.html.slim]: I don't known about `=<` and `=>`. HEHE..

Comment (neves) [inline app/views/welcome/index.html.slim]: https://github.com/slim-template/slim#trailing-and-leading-whitespace--
It's ugly as hell, but...

**Pull request merged**